packages/cubejs-testing/test/smoke-redshift.test.ts

```diff
@@ -28,7 +33,30 @@ describe('redshift', () => {
 
   afterAll(async () => {
     await birdbox.stop();
+    Object.keys(env).forEach((key) => {
+      delete process.env[key];
+    });
   });
 
   test('query measure', () => testQueryMeasure(client));
-});
+};
+
+// Base environment vars that are required if S3 export is to be enabled.
+const baseS3Env: { [ key: string ]: string } = {
+  CUBEJS_DB_EXPORT_BUCKET_TYPE: 's3',
+  CUBEJS_DB_EXPORT_BUCKET: 'foo-bucket',
+  CUBEJS_DB_EXPORT_BUCKET_AWS_REGION: 'bar-region'
+};
+// No export/unload configured
+describe('redshift', reusableSuite());
```

Comment:
Perhaps I'm missing something obvious, but how can one override the actual values of the env variables when running the tests?

Reply:
They can't.  I didn't see that you were actually testing the export/unload process, so I did not think the tests needed to do this (unless I missed something). The specific issue I had originally, was the connection process failing, simply because key/secret were not provided in the config, so these tests pass by virtue of the proper configuration being present.

Configuring the test in this manner, runs the 3 separate cases of configuration.  Per your note above, there are 4 cases (when the ARN and Keys are provided), but I will need to add those conditions in.